packages/lit-element/src/lib/css-tag.ts

```diff
@@ -94,5 +92,12 @@ export const css = (
     (acc, v, idx) => acc + textFromCSSResult(v) + strings[idx + 1],
     strings[0]
   );
-  return new CSSResult(cssText, constructionToken);
+  let result = cssResultCache.get(cssText);
+  if (!result) {
```

Comment:
```suggestion
  if (result === undefined) {
```

to avoid ToBoolean conversions

Reply:
Done (more bytes).

Reply:
I know it's a few more bytes, but done repeatedly it compresses well, and it generates fewer byte codes. That won't matter here, but maybe in in hot paths like hasChanged. I like to be consistent.